hadoop/util.sh

```diff
@@ -73,3 +73,36 @@ function use_java8() {
    restart_worker_services
  fi
 }
+
+# Wait until namenode is out of safe mode. Takes 2 params
+# first : Number of attempts function will make to get namenode out of safemode
+# second : Number of seconds each attempt will sleep for waiting for namenode to come out of sleep mode
+function wait_until_namenode_out_of_safe() {
+    n=0
+    num_attempts=$1
```

Comment:
We should have a default value for these fields - say 5s for sleep and 50 for number of retries?